ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -1793,17 +1793,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity
-- as: gcp-ipi-proxy-oidc-arm-f28
+- as: gcp-ipi-proxy-oidc-filestore-csi-arm-f28
```

Comment:
FYI we had updated `gcp-ipi-proxy-oidc-arm-f28` into `gcp-ipi-proxy-oidc-mini-perm-arm-f28`, in https://github.com/openshift/release/pull/62332.